readme.md

```diff
@@ -1,30 +1,51 @@
 # Prototype Playground
 What is it? This is a base environment, with the minimum amount of setup and dependencies to start building web prototypes.
 
+You'll be setting up Node, a Javascript environment that lets you code on your local machine. With Node, you can use NPM and Gulp. NPM is a package manager that lets you download useful tools to help you code. Gulp is a task runner that automates whatever you tell it to. It's commonly used to check for errors, compile your code, and auto refresh your browser.
+
 # Getting started
-Clone this repository, then:
+1. Install [Atom](https://atom.io/) or another text editor of your choice
+2. Download XCode from the App Store
+3. Open the Terminal app
+4. Install Homebrew. This makes it [easier to install Node](http://blog.teamtreehouse.com/install-node-js-npm-mac ).
+    1. In Terminal, type ``ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ``
+    2. You’ll see messages in the Terminal explaining what you need to do to complete the installation process.
+5. Install Node and NPM
+    1. Once Homebrew is installed, type ```brew install node```
+    2. To make sure Node is installed, type `node -v`. You should see the version that's installed, ex:  `v0.10.31`.
+    3. Make sure NPM is installed by typing `npm -v`
+6. Install Gulp globally
+    1. Type `npm install -g gulp`
+7. Create a folder for your new project and navigate to it in Terminal
+    1. You can use the `cd` command. For example: `cd /Applications/XAMPP/xamppfiles/htdocs/my-project`
+    2. You can also drag your folder onto the Terminal app in your dock.
+8. Create a package.json file for your project. This will keep track of dependencies.
+    1. Type `npm init` and answer the questions it asks you
+9. Install Gulp locally
+    1. Type `npm install --save-dev gulp`. The `--save-dev` bit saves gulp as a dependency in your package.json file.
+10. Download the prototype playground and place it in your project folder
+    1. Click the green Clone or Download button at the top of this page and download the ZIP.
+11. Download Node modules
+    1. In Terminal type `npm install`.
+12. Initialize Gulp
+    1. Type `gulp`.
+13. Open your project folder in your text editor and start coding!
 
-1. install dependencies. (If you don't have npm installed, see the Prerequisites section)
-    ```
-    npm install
-    ```
-2. Build, start the development server, and start watchers with:
-    ```
-    npm run start
-    ```
-    This will start a development server at `localhost:3000`. The console will also export a "external" ip address that can be accessed from other devices if applicable.
```

Comment:
Somewhere we will need to note where the site is accessible, even though it does say in  terminal.